File: AppMobile/Models/ActivityModel.cs
using SQLite;

namespace AppMobile.Models
{
    class ActivityModel
    {
        [PrimaryKey]
        public string id { get; set; }
        public string name { get; set; }
        public string route { get; set; }
        public string date { get; set; }
        public string duration { get; set; }
        public int kilometers { get; set; }
        public string type { get; set; }
        public string athleteUsername { get; set; }
    }
}

File: WebPages/Vista-Deportista/src/app/Models/activity-model.ts
/**
 * Model for the Activity Table
 */
export interface ActivityModel {
    id: string,
    name: string,
    route: string, // Check back which type will the route be
    date: string,
    duration: string, 
    kilometers: number,
    type: string,
    athleteUsername: string
}
